fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -2594,6 +2594,7 @@ ACTOR Future<Void> reply(CommitBatchContext* self) {
 		state Optional<std::set<Tag>> writtenTags;
 		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
 			writtenTags = self->writtenTags;
+			// ASSERT(writtenTags.get().size() > 0);
```

Comment:
Probably this needs a comment explaining why this assertion needs to hold true in all scenarios.

Reply:
typo in the draft